### PR TITLE
[Enhancement] update warehouse status

### PR DESCRIPTION
**Pull request by yandongxiao**
export args for operator helm chart and update warehouse status

### CODE REVIEW

File: helm-charts/charts/kube-starrocks/charts/operator/values.yaml [hunk 1]
```diff
@@ -33,6 +33,8 @@ starrocksOperator:
     # image sliced by "repository:tag"
     repository: starrocks/operator
     tag: v1.8.6
+  args:
```
Comment (kevincai) [inline helm-charts/charts/kube-starrocks/charts/operator/values.yaml:36]: not necessary?

Comment (yandongxiao) [reply helm-charts/charts/kube-starrocks/charts/operator/values.yaml:36]: Operator has some args, like -v for debug log, --namespace

Comment (kevincai) [reply helm-charts/charts/kube-starrocks/charts/operator/values.yaml:36]: I think the namespace has already exposed?. debug log is just for troubleshoot? the example --elect-leader is confusing.

Comment (yandongxiao) [reply helm-charts/charts/kube-starrocks/charts/operator/values.yaml:36]: We can hide the --elect-leader in template.

Comment (yandongxiao) [reply helm-charts/charts/kube-starrocks/charts/operator/values.yaml]: I have remove the args from values.yaml

**Review by kevincai — APPROVED** [review]

**Pull request merged**